tests/integration_tests/modules/test_apt.py

```diff
@@ -78,6 +78,16 @@
         pb0uBy+g0oxJQg15
         =uy53
         -----END PGP PUBLIC KEY BLOCK-----
+    test_write:
```

Comment:
I don't think we actually document expected behavior for reusing keys (using `test_write` twice), which gives me some hesitations about depending on this behavior. 

Also, since `sources` in the jsonschema is of type object (which roughly maps to a python `dict`), won't this map to a single key/value pair in a dict? If so, I don't think this test does what was intended and should probably be updated accordingly. (this should probably be documented better)